transactions/transactions_service.go

```diff
@@ -836,40 +795,6 @@ func makePreimageHex() ([]byte, error) {
 	return bytes, nil
 }
 
-func (svc *transactionsService) parseMetadata(metadata lnclient.Metadata) ([]byte, error) {
```

Comment:
Why are we removing this, we decided not to decode it on the fly right? Or are you pushing another change which does this during db save?

Reply:
We don't decode boostagrams on the fly. This is the metadata - which we also do not decode on the fly (it's returned as-is in NWC requests)

Reply:
The frontend does not use the `metadata`.

Reply:
Ah okay, then it should be simpler 👍